src/test/kotlin/blackjack/domain/CardNumberCalculatorTest.kt

```diff
@@ -52,7 +51,7 @@ class CardNumberCalculatorTest {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = ["ONE", "TWO", "THREE", "FOUR", "FIVE", "SIX", "SEVEN", "EIGHT", "NINE", "TEN"])
+    @ValueSource(strings = ["TWO", "THREE", "FOUR", "FIVE", "SIX", "SEVEN", "EIGHT", "NINE", "TEN"])
```

Comment:
```suggestion
    @EnumSource(value = CardNumber::class)
```
EnumSource를 활용하시면 전체 enum을 불러올 수 있어요!